VictorPOP/tema4-DICT-VictorPOP.py

```diff
@@ -0,0 +1,120 @@
+import pandas as pd
+
+'''
+Using data from Eurostat, create a list of tuples representing the
+"Self-perceived health by country and sex, age group >16, for people
+living in cities" for 2017-2018.
+Have at least 30 valoares in your dataset.
+I exported the data needed from eurostat and deleted the information
+we dont need then I modified the CSV file so that the lack of valoare
+':' was replaced with 0
+note: only "very good" perception was used.
+The dataset will have the following structure:
+[(country, year, sex, health_index) ]
+'''
+
+datainit = pd.read_csv('hlth_silc_01_1_Data.csv')
+datasect = pd.DataFrame(datainit, columns=['GEO', 'TIME', 'SEX', 'Value'])
+datalist = datasect.to_numpy().tolist()
+print(datalist)
+input()
+
+# two dicts that group all data by country for each year
+# health_index_2017 ={'France': [sex, health_index]}
+
+# 1st solution
+health_index_2017 = {
+    geo+'_'+sex: [valoare]
+    for geo, timp, sex, valoare in datalist
+    if timp == 2017
+}
+health_index_2018 = {
+    geo+'_'+sex: [valoare]
```

Comment:
Now that we know how to properly concatenate strings, please update the implementation and fix this

Reply:
Done, I didn't revisit the code till now... now I did